src/udev/ata_id/ata_id.c

```diff
@@ -291,9 +275,7 @@ static void disk_identify_fixup_string(
 }
 
 static void disk_identify_fixup_uint16 (uint8_t identify[512], unsigned offset_words) {
-        uint16_t *p;
-
-        p = (uint16_t *) identify;
+        uint16_t *p = (uint16_t*) identify;
```

Comment:
this is an alignment issue btw. identitfy is byte-aligned, and we read it as 16bit value.

maybe rework this to use unaligned_read_u16()?